docs/sphinx/using/tutorials.rst

```diff
@@ -0,0 +1,13 @@
+*************************
+CUDA Quantum Tutorials
+*************************
+
+Tutorials that give an in depth view of CUDA Quantum and its applications in Python.
+
+.. nbgallery::
+   /examples/python/tutorials/executing_circuits.ipynb
```

Comment:
I think you should simply be able to indent these 4 or more whitespaces for the spellcheck errors to go away.